packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/Unbranded-TypeSpec.tsp

```diff
@@ -140,6 +146,36 @@ model ModelWithRequiredNullableProperties {
   requiredFixedEnum: StringFixedEnum | null;
 }
 
+@doc("This is a base model without any required properties")
```

Comment:
we should try to avoid adding more to the library, we should use unit tests instead